src/Paysera/WalletApi/Entity/ClientWalletPermissions.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+/**
+ * Paysera_WalletApi_Entity_ClientWalletPermissions
+ */
+class Paysera_WalletApi_Entity_ClientWalletPermissions
+{
+    /**
+     * Scopes
+     */
+    const SCOPE_BALANCE = 'balance';
+    const SCOPE_STATEMENTS = 'statements';
+
+    /**
+     * @var int
+     */
+    protected $walletId;
+
+    /**
+     * @var string
+     */
+    protected $accountNumber;
+
+    /**
+     * @var array
+     */
+    protected $scopes = array();
+
+    /**
+     * @return Paysera_WalletApi_Entity_ClientWalletPermissions
+     */
+    public static function create()
+    {
+        return new self();
+    }
+
+    /**
+     * @param int $walletId
+     * @return Paysera_WalletApi_Entity_ClientWalletPermissions
+     */
+    public function setWalletId($walletId)
```

Comment:
it doesn't look like Paysera_WalletApi_Entity_ClientWalletPermissions can be without walletId, so it must be in the constructor, otherwise, it can be, that we would have permissions, but no id set...

by the way, if we would be just reusing what we have, maybe we could survive without constructor, as it's already like that...
Paysera_WalletApi_Entity_Wallet

Reply:
Done